client/blocks/login/social.jsx

```diff
@@ -74,12 +82,30 @@ class SocialLoginForm extends Component {
 		this.recordEvent( 'calypso_login_social_button_click' );
```

Comment:
Not related to this pull request, but we should log the `social_account_type` property with this event.